README.md

```diff
@@ -1,29 +1,68 @@
-# Gnareact
```

Comment:
Should this Readme say something about what this repo is for? I can't tell from here. Wasn't it something with Youtube video peeking or something?